models/ed/R/met2model.ED2.R

```diff
@@ -119,20 +119,19 @@ met2model.ED2 <- function(in.path, in.prefix, outfolder, start_date, end_date, l
     SW   <- ncvar_get(nc, "surface_downwelling_shortwave_flux_in_air")
     LW   <- ncvar_get(nc, "surface_downwelling_longwave_flux_in_air")
     CO2  <- try(ncvar_get(nc, "mole_fraction_of_carbon_dioxide_in_air"), silent = TRUE)
-    
+
     useCO2 <- is.numeric(CO2)
-    
+
     ## convert time to seconds
     sec <- udunits2::ud.convert(sec, unlist(strsplit(nc$dim$time$units, " "))[1], "seconds")
-    
+
     ncdf4::nc_close(nc)
-    
-    dt <- ifelse(lubridate::leap_year(year) == TRUE, 
-                 366 * 24 * 60 * 60 / length(sec), # leap year
-                 365 * 24 * 60 * 60 / length(sec)) # non-leap year
-    
+
+    diy <- PEcAn.utils::days_in_year(year)
+    dt <- diy * 24 * 60 * 60 / length(sec)
```

Comment:
use ud.convert? This pattern seems to be repeated a lot. Maybe we need another function `seconds_in_year`?